File: 0867-transpose-matrix/0867-transpose-matrix.java
class Solution {
    public int[][] transpose(int[][] matrix) {
        int result[][]=new int[matrix[0].length][matrix.length];
        for(int e=0;e<matrix.length;e++){
            for(int f=0;f<matrix[0].length;f++){
                result[f][e]=matrix[e][f];
            }
        }
        return result;
    }
}

File: 0867-transpose-matrix/0867-transpose-matrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
        int row=matrix.size();
        int col=matrix[0].size();
        vector<vector<int>> result(col);
        for(int e=0;e<row;e++){
            for(int f=0;f<col;f++){
                result[f].push_back(matrix[e][f]);
            }
        }
        return result;
    }
};